metals/src/main/scala/scala/meta/internal/metals/debug/DebugProvider.scala

```diff
@@ -571,29 +589,18 @@ class DebugProvider(
   /**
    * Validate if build target provided in params exists.
    * On the contrary, test classes aren't validated.
-   *
-   * If build tool doesn't support test selection fallback to the already
-   * defined and supported SCALA_TEST_SUITES request kind.
    */
   def resolveTestSelectionParams(
       request: ScalaTestSuitesDebugRequest
   )(implicit ec: ExecutionContext): Future[b.DebugSessionParams] = {
     val makeDebugSession = () =>
       buildTargets.info(request.target) match {
         case Some(buildTarget) =>
-          val debugSession =
-            if (supportsTestSelection(request.target))
```

Comment:
Are we sure that test selection is supported everywhere? Mill BSP is still missing it, no?

Reply:
Only sbt and Bloop support debugging thus can reach that code. Mill doesn't implement [debug capability](https://github.com/com-lihaoyi/mill/blob/252407daba0acc4b93f34e2e4809522ea18760fa/scalalib/src/mill/scalalib/bsp/BspModule.scala#L26) so it's not a problem. If execution reached here, then it will fail no matter if it's supported or not. 

However, I think that before this PR will be merged, https://build-server-protocol.github.io/docs/specification.html#debug-request should be updated (or at least there should be a PR) with all mandatory kinds that build server needs to implement. Currently, one has to search for debug's data kinds in code which is a bit unfortunate. What do you think of that? Will it be a decent solution to your concerns?

cc: @ckipp01 as you had similar doubts as Tomasz.

Reply:
Yes, it makes sense. Thanks!

Reply:
here's PR which does that https://github.com/build-server-protocol/build-server-protocol/pull/383, could you take a look at it to ocnfirm that I didn't mess anything? :D